.github/workflows/main.yaml

```diff
@@ -37,11 +37,11 @@ on:
         type: boolean
       runner_template:
         description: Runner template to use
-        default: hosted-prov-e2e-ci-runner-spot-x86-64-template-n2-highmem-32-v2
+        default: hosted-prov-e2e-ci-runner-spot-n2-highmem-16-gl-template-v1
         type: string
       zone:
         description: GCP zone to host the runner
-        default: us-west1-b
+        default: asia-south2-c
```

Comment:
Since the instance template is present in that region only, can we avoid providing option to user to select the zone.
https://github.com/rancher/hosted-providers-e2e/actions/runs/7397821063/job/20125653539#step:6:31